packages/commerce-events-collectors/src/utils/aep/order.ts

```diff
@@ -1,7 +1,24 @@
-import * as sdkSchemas from "@adobe/commerce-events-sdk";
+import * as sdkSchemas from "@adobe/magento-storefront-events-sdk/dist/types/types/schemas";
 
 import { Order, Payment } from "../../types/aep";
 
+const getAepPaymentCode = (paymentMethodCode: string) => {
```

Comment:
How this change is related to email address?

Reply:
This is just to keep the code in commerce-events and storefronts packages in sync.  Using the AEP payment codes was never included in commerce events, but it should be and I have included it now because of the inclusion/addition of the unit test which expects this logic.